### DISCUSSION TITLE
courses/mappers/18

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#181] [Ideas]
# courses/mappers/18

SAF CLI Integration The following is a supplemental lesson on how to integrate your developed mapper with the SAF CLI. Integration with the SAF CLI will allow your mapper to be ...

https://deploy-preview-155--mitre-saf-training.netlify.app/courses/mappers/18.html

**Comment by andytang99** [comment]
Skeleton Convert Command File has extra period "const converter = new Mapper(input, flags.['with-raw'])" on line 24. Should be "const converter = new Mapper(input, flags['with-raw'])". This is located in Integration step 1.
